### ISSUE TITLE
release for 4.3.0

### ISSUE THREAD

**Issue by zemu-csi** [BCLCommons/bcl#2]
Could you create a release tarball for 4.3.0? This would help a lot for packaging.

**Comment by brownbp1** [comment]
Hey @zemu-unile - sure thing, that should be no problem. Give me a day or two - I want to see if we will be able to get up pre-compiled binaries along with the source code tarball. Our previous release scripts were tied to our old license model, so we need to refactor them. If it looks like it'll be longer than that then I will just upload the compressed source code in the meantime.

Thanks for your interest.

**Comment by brownbp1** [comment]
Checking back in - I have been a bit strapped for time, and it does turn out that refactoring our scripts to correctly build binaries without any license restriction nonsense will require some thought. So! In the meantime, I attached the source code in a tarball as requested. I hope this helps. 

We've got a few other interesting developments that me or someone else will eventually post about in the Discussions board. In the meantime, if any other issues arise just submit and Issue and we'll take a look. 

Thanks again.
